src/EFCore.PG.Trgm/Extensions/NpgsqlTrgmDbFunctionsExtensions.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+
+namespace Microsoft.EntityFrameworkCore
+{
+    public static class NpgsqlTrgmDbFunctionsExtensions
+    {
+        /// <summary>
+        /// show_trgm(text)
```

Comment:
Probably, these trgm function names should be moved to remarks. Maybe something like this should be:

```csharp
/// <remarks>
/// The method call is translated to <c>show_trgm(text)</c>.
/// </remarks>
```

Reply:
Is acdf25116ce0677eb0ada147c787276193349ce3 sufficient for this ?

Reply:
Sure, thanks!